exporter.py

```diff
@@ -105,10 +105,15 @@ def initialize_collector_by_name(class_name, logger):
         return None
 
 
-def get_targets():
-    request = requests.get(url="http://" + os.environ['INVENTORY'] + "/vrops_list")
-    targets = request.json()
-    return targets
+def get_targets(inventory):
+    try:
+        request = requests.get(url="http://" + os.environ['INVENTORY'] + "/vrops_list")
+        targets = request.json()
+        return targets
+    except requests.exceptions.ConnectionError as e:
+        logger.critical(f'No connection to {inventory} - Error: {e}')
+        logger.critical(f'Exit')
+        sys.exit(0)
```

Comment:
Just to get this right, are you doing this to enforce a pod restart?

Reply:
That is a good remark. What is the desired behaviour?

Reply:
Right now the pod would restart, which is not too bad, since it can't do anything anyway.
I would say let it retry 5 times with a 10seconds timeout and then exit.